client/bootstrap

```diff
@@ -36,7 +36,8 @@
 
 import argparse
 import datetime
-import imp
+import importlib.machinery
+import importlib.util
 import json
 import jsonrpclib
```

Comment:
Do we still need to address replacing this and sleekxmpp based on this output from the email thread?

ZeroTouch provisioning bootstrap scripts using 'import jsonrpclib' and
'import sleekxmpp' in python2 will fail. Use python3 instead.